### PR TITLE
Update problem-1.cpp

### PR DESCRIPTION
**Pull request by SohomGhosh10**
Fixed one error in the code where result value is initialized with n - 1. 
Changed that value to -1.

### CODE REVIEW

File: binary-search/problem-1.cpp [hunk 1]
```diff
@@ -22,22 +22,22 @@ int main()
 
 int findTotalOccurrence(int arr[], int n, int key, bool check)
 {
-    int low = 0, high = n - 1, result = n - 1;
+    int low = 0, high = n - 1, result = -1;
```
Comment (AvikAgarwala) [inline binary-search/problem-1.cpp:25]: Removed blank space after - (minus).

Comment (SohomGhosh10) [reply binary-search/problem-1.cpp:25]: Ok

**Comment by AvikAgarwala** [comment]
@Alik-Agarwala kindly check this code pushed by you. However, @SohomGhosh10 fixed it.

**Comment by Alik-Agarwala** [comment]
Thanks @SohomGhosh10 for it. Mistakenly, it was done.

**Pull request merged**